engines/python/setup/djl_python/rolling_batch/rolling_batch.py

```diff
@@ -267,24 +273,26 @@ def reset(self):
         self.req_id_counter = 0
 
     @abstractmethod
-    def inference(self, input_data, parameters):
+    def inference(self, input_data, parameters, adapters=None):
```

Comment:
do we have scenarios where some input doesn't need adapter and some needs. How do we get the order of input data with adapters?